pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -140,9 +140,6 @@ func getFloatingIPByPortID(client *gophercloud.ServiceClient, portID string) (*f
 			return false, err
 		}
 		floatingIPList = append(floatingIPList, f...)
-		if len(floatingIPList) > 1 {
```

Comment:
@anguslees Thank for your review very much.
My modify just prevent it from checking {len(floatingIPList) > 1} again, and I see your comment, I will check it in loop and don't check it at line 157.